front/app/resources/GetGlobalPermissions.tsx

```diff
@@ -28,6 +28,11 @@ export type GetGlobalPermissionsChildProps =
   | null
   | Error;
 
+// TODO remove this later when we actually start using 'visiting' as a permission
+const notVisitingPermission = (data: IGlobalPermissionData) => {
+  return (data.attributes.action as any) !== 'visiting';
+};
+
```

Comment:
For my own knowledge - what was this 'visiting' permission we're adding again? When would it be used?